kotlin/app/src/main/java/com/google/android/gms/samples/wallet/viewmodel/CheckoutViewModel.kt

```diff
@@ -0,0 +1,71 @@
+package com.google.android.gms.samples.wallet.viewmodel
+
+import android.app.Application
+import android.util.Log
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.google.android.gms.common.api.ApiException
+import com.google.android.gms.samples.wallet.util.PaymentsUtil
+import com.google.android.gms.tasks.Task
+import com.google.android.gms.wallet.IsReadyToPayRequest
+import com.google.android.gms.wallet.PaymentData
+import com.google.android.gms.wallet.PaymentDataRequest
+import com.google.android.gms.wallet.PaymentsClient
+
+class CheckoutViewModel(application: Application) : AndroidViewModel(application) {
+
+    /**
+     * A client for interacting with the Google Pay API.
+     *
+     * @see [PaymentsClient](https://developers.google.com/android/reference/com/google/android/gms/wallet/PaymentsClient)
+     */
+    private val paymentsClient: PaymentsClient = PaymentsUtil.createPaymentsClient(application)
+
+    // LiveData with the result of whether the user can pay using Google Pay
+    private val canUseGooglePayData: MutableLiveData<Boolean> by lazy {
```

Comment:
This is a backing field, and according to the Kotlin naming convention (https://kotlinlang.org/docs/coding-conventions.html#names-for-backing-properties),  `_canUseGooglePlayData` seems to be following the naming convention.